packages/db/tests/search/test-StatsSearch.js

```diff
@@ -0,0 +1,390 @@
+'use strict';
+
+const test = require('ava');
+const cryptoRandomString = require('crypto-random-string');
+const range = require('lodash/range');
+const { StatsSearch } = require('../../dist/search/StatsSearch');
+
+const {
+  destroyLocalTestDb,
+  generateLocalTestDb,
+  GranulePgModel,
+  CollectionPgModel,
+  fakeCollectionRecordFactory,
+  fakeGranuleRecordFactory,
+  fakeProviderRecordFactory,
+  migrationDir,
+  fakePdrRecordFactory,
+  fakeExecutionRecordFactory,
+  PdrPgModel,
+  ExecutionPgModel,
+  ProviderPgModel,
+} = require('../../dist');
+
+const testDbName = `collection_${cryptoRandomString({ length: 10 })}`;
+
+test.before(async (t) => {
+  const { knexAdmin, knex } = await generateLocalTestDb(
+    testDbName,
+    migrationDir
+  );
+
+  t.context.knexAdmin = knexAdmin;
+  t.context.knex = knex;
+
+  t.context.collectionPgModel = new CollectionPgModel();
+  t.context.granulePgModel = new GranulePgModel();
+  t.context.providerPgModel = new ProviderPgModel();
+  t.context.PdrPgModel = new PdrPgModel();
+  t.context.ExecutionPgModel = new ExecutionPgModel();
+
+  const statuses = ['queued', 'failed', 'completed', 'running'];
```

Comment:
example error:
{
  "Cause": "{\"errorType\":\"CumulusMessageAdapterExecutionError\",\"errorMessage\":\"Traceback (most recent call last):\\n  File \\\"/usr/lib64/python2.7/runpy.py\\\", line 165, in _run_module_as_main\\n    mod_name, loader, code, fname = _get_main_module_details(_Error)\\n  File \\\"/usr/lib64/python2.7/runpy.py\\\", line 133, in _get_main_module_details\\n    return _get_module_details(main_name)\\n  File \\\"/usr/lib64/python2.7/runpy.py\\\", line 119, in _get_module_details\\n    code = loader.get_code(mod_name)\\n  File \\\"/usr/lib64/python2.7/pkgutil.py\\\", line 281, in get_code\\n    self.code = compile(source, self.filename, 'exec')\\n  File \\\"/opt/__main__.py\\\", line 84\\n    sys.stderr.write(f'warning setting command to {command}\\\\n')\\n                                                             ^\\nSyntaxError: invalid syntax\\n\",\"trace\":[\"CumulusMessageAdapterExecutionError: Traceback (most recent call last):\",\"  File \\\"/usr/lib64/python2.7/runpy.py\\\", line 165, in _run_module_as_main\",\"    mod_name, loader, code, fname = _get_main_module_details(_Error)\",\"  File \\\"/usr/lib64/python2.7/runpy.py\\\", line 133, in _get_main_module_details\",\"    return _get_module_details(main_name)\",\"  File \\\"/usr/lib64/python2.7/runpy.py\\\", line 119, in _get_module_details\",\"    code = loader.get_code(mod_name)\",\"  File \\\"/usr/lib64/python2.7/pkgutil.py\\\", line 281, in get_code\",\"    self.code = compile(source, self.filename, 'exec')\",\"  File \\\"/opt/__main__.py\\\", line 84\",\"    sys.stderr.write(f'warning setting command to {command}\\\\n')\",\"                                                             ^\",\"SyntaxError: invalid syntax\",\"\",\"    at Interface.<anonymous> (/var/task/webpack:/node_modules/@cumulus/cumulus-message-adapter-js/dist/cma.js:155:1)\",\"    at Interface.emit (events.js:326:22)\",\"    at Interface.EventEmitter.emit (domain.js:483:12)\",\"    at Interface.close (readline.js:416:8)\",\"    at Socket.onend (readline.js:194:10)\",\"    at Socket.emit (events.js:326:22)\",\"    at Socket.EventEmitter.emit (domain.js:483:12)\",\"    at endReadableNT (_stream_readable.js:1241:12)\",\"    at processTicksAndRejections (internal/process/task_queues.js:84:21)\"]}",
  "Error": "CumulusMessageAdapterExecutionError",
  "failedExecutionStepName": "MoveGranuleStep"
}